spec/features/edit_user_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+describe 'the edit user details process', type: :feature do
```

Comment:
Tests looks good, possibly you could add a space between the actions and the expect assertions just to format it a bit better.